default.go

```diff
@@ -84,7 +87,7 @@ func (d *DefaultNode) setDefaults(fields models.Fields, tags models.Tags) (model
 	newTags := tags
 	tagsCopied := false
 	for tag, value := range d.d.Tags {
-		if _, ok := tags[tag]; !ok {
+		if v := tags[tag]; v == "" {
```

Comment:
> The reason being that it's not possible to write a point in line protocol with an empty tag value 

I didn't realize this. Since it is true, (I just tested it) it is not possible that someone was relying on the subtle difference in behavior. 

So lets make the change as is but not mark it as BREAKING since it is not actually possible that someone was relying on the difference.